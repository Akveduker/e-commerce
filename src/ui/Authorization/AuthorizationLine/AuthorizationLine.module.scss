@import "../../../styles/_vars.scss";
.line {
    text-align: center;
    font-size: 18px;
    position: relative;
    &::after,
    &::before {
        z-index: -1;
        content: "";
        display: block;
        width: 100%;
        height: 1px;
        background-color: $C1B;
        position: absolute;
        top: 0;
        bottom: 0;
        margin: auto 0;
    }
    &::after {
        right: 0;
    }
    &::before {
        left: 0;
    }
}
.text {
    display: inline-flex;
    background-color: $C1J;
    padding: 0 5px;
}
