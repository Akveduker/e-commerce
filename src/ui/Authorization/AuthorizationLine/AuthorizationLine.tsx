import React from 'react';
import s from './AuthorizationLine.module.scss'
const AuthorizationLine = () => {
    return (
        <p className={s['line']}>
            <span className={s['text']}>ИЛИ</span>
        </p>
    );
};

export default AuthorizationLine;