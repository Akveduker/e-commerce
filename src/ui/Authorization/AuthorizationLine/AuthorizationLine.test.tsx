import { render } from "@testing-library/react"
import AuthorizationLine from "./AuthorizationLine"
describe('AuthorizationLine', () => {
    test('Снапшот', () => {
        const { container } = render(<AuthorizationLine />)
        expect(container).toMatchSnapshot()
    })
})