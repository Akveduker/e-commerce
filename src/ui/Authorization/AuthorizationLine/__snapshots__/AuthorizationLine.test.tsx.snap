// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthorizationLine Снапшот 1`] = `
<div>
  <p
    class="line"
  >
    <span
      class="text"
    >
      ИЛИ
    </span>
  </p>
</div>
`;
